### DISCUSSION TITLE
Integrate Full Site Editing into WP Rig

### DISCUSSION THREAD

**Discussion by dthenley** [wprig/wprig#696] [Ideas]
Full site editing would be an extremely useful for both developer and user, so offering that would show more of an advantage for any user wanting to use WPRig for its forward thinking approach.

You can read more about what it is and how to get started with it here https://gutenbergtimes.com/full-site-editing/

**Comment by LukaszJaro** [comment]
Good point, this is the future!

I'm developing a website with wprig 2.0 for the first time. Right of the bat, the header design is not what is most commonly used. It's more of a mobile first approach, where I'm looking for a typical layout with logo and nav inlined. So yes, I can use grid or flexbox to set that up, but would be nice to have easily interchangeable headers which I would think is possible with full site editing.

**Reply by robruiz** [reply]
Keep in mind that WP Rig is designed to generate a starter theme for theme developers. Dictating the nav style is more of a concern for the theme dev and shouldn't be opinionated by the framework itself. With that said, I agree it would be nice if there was an easy way to implement a best practice drop in solution for common nav configurations. I'm not sure relying on full-site editing to solve this problem makes a lot of sense. WP Rig is not designed to be used right out of the box. It's a starting point for a parent theme, not a theme itself. It's important to always be cognitive of this when thinking about improvements to it. It should not be very opinionated.
